docs/general-settings.md

```diff
@@ -20,3 +21,4 @@ settings: {
 * **header.title** defines the top left navigation title.
 * **header.favicon** defines the favicon. It rquires a standard favicon file with the `.ico` extension, and 16x16px or 32x32px dimensions.
 * **responsiveNavigation** allows adding a button on the left side of the top navigation. Upon click, the button shows or hides the left navigation. The possible values are `simple` and `simpleMobileOnly`. `simple` displays the button regardless of the browser window´s size, while `simpleMobileOnly` shows the button when the width is lower than `600px`. If you don't specify any value for  **responsiveNavigation**, the button will not be displayed. The same applies when you enable **hideSideNav** for the current active navigation node. 
+* **sideNavFooterText** a string that is displayed in a sticky footer inside Side Navigation. Good place to display the version of your application.
```

Comment:
It can also be a function (not async) since we are using `getConfigValue` to fetch it. Do we want to mention this?

Reply:
Good point, I'll mention it.

Reply:
Sadly, it can't be. Checked with arrow and normal function.
![image](https://user-images.githubusercontent.com/22825433/56965170-f0a4b480-6b5c-11e9-93d3-49516d61cf77.png)